scripts/cnn_feat_extraction/README.md

```diff
@@ -12,17 +12,57 @@
 
 To extract the CNN feature vectors for a set of images:
 
-1. Convert images to size 227x227 pixels.  For example, using ImageMagick:
+### 1. Preprocessing
+
+Convert images to size 227x227 pixels.  For example, using ImageMagick:
 
 `$ mogrify -path imageset/# -format jpg -resize "227x227^" -gravity center -crop 227x227+0+0 +repage imageset/#/*.jpg`
 
-A future update will perform this preprocessing within the `feat_csv.py` script.
+Performing this preprocessing separately is strongly recommended, as the caffe resizing can be quite slow.
+
+### 2. Configuration File
+
+Create a config file to specify locations of the trained CNN (Caffe model), the image files, and other parameters.  You can use `example.cfg` as a starting point.
+
+*[Files]*
+- `model_dir` is the folder where caffe model definitions are stored.
+- `imageset_dir` is the imageset directory. It contains images or subdirectories. Do not include non-image files in the directory; while it will be rejected, it will mess up the imagecount.
```

Comment:
Suggest changing "it will be rejected" to "they will be rejected" and "it will mess up the imagecount" to "the total imagecount will be incorrect."

Reply:
"mess up" is not very specific, is it...